tools/expat/Makefile

```diff
@@ -9,21 +9,25 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=expat
 PKG_CPE_ID:=cpe:/a:libexpat:expat
-PKG_VERSION:=2.2.10
+PKG_VERSION:=2.4.6
```

Comment:
Btw, I noticed mismatch in the commit subject and with this version

``2.4.1`` vs ``2.4.6``

Reply:
Fixed.

Reply:
:+1: